app/models/health/team/member.rb

```diff
@@ -122,10 +122,29 @@ def in_use?
 
     def email_domain_if_present
       return if email.blank?
-      unless Health::Agency.email_valid?(email)
-        errors.add(:email ,'address must match a provider organization domain (e.g. @tuftsmedical.org — can\'t be @gmail.com or other generic domain)')
-      end
+
+      errors.add(:email, 'address must match a provider organization domain (e.g. @tuftsmedical.org — can\'t be @gmail.com or other generic domain)') unless Health::Agency.email_valid?(email)
     end
 
+    def self.as_health_contacts(_force = false)
+      contact_columns = {
+        patient_id: :patient_id,
+        source_id: :id,
+        source_type: :type,
+        description: :title,
+        phone: :phone,
+        email: :email,
+        collected_on: :date_of_last_contact,
+      }.invert
+
+      Health::Team::Member.all.map do |row|
```

Comment:
Do you maybe want to find these in batches?  I think there could be 10s of thousands of these potentially.